apps/ledger-live-desktop/src/renderer/components/SyncOnboarding/DeviceConnection/Searching.tsx

```diff
@@ -21,6 +23,14 @@ const SyncOnboardingDeviceConnectionSearching = ({
   const history = useHistory();
   const { setDeviceModelId } = useContext(OnboardingContext);
   const theme = useTheme();
+  const currentDevice = useSelector(getCurrentDevice);
+
+  useEffect(() => {
+    if (currentDevice && currentDevice.modelId !== deviceModelId) {
+      setDeviceModelId(currentDevice.modelId);
+      history.replace("/onboarding/select-use-case");
```

Comment:
On the demo video it does not seem to redirect automatically to the device model selection of the onboarding - the user has to click on the exit button. Is it normal ?

Reply:
During the demo I clicked on the exit button in order to skip the install apps step during onboarding. 
It's not related to this PR and the `history.replace` is used here to replace the stax onboarding page by the LNX one

Reply:
Misunderstanding on the demo video hehe